llvm/test/CodeGen/X86/patchable-prologue.ll

```diff
@@ -193,3 +193,20 @@ do.body:                                          ; preds = %do.body, %entry
 do.end:                                           ; preds = %do.body
   ret void
 }
+
+
+; Test that inline asm is properly hotpatched. We currently don't examine the
+; asm instruction when printing it, thus we always emit patching NOPs.
+
+; 64: inline_asm:
```

Comment:
Adding a 32 bit version for completion would be nice, unless it is intentional?

Reply:
Yes, it is intentional since I thought it doesn't bring value. The issue covered is not dependent of the specific architecture.